README.md

```diff
@@ -1 +1,92 @@
+| BACKEND_HOSTNAME     | http://127.0.0.1:4000/api  | URL used to connect to backend server       |
+| BACKEND_USERNAME     |                            | User name used to connect to backend server |
+| BACKEND_PASSWORD     |                            | Password used to connect to backend server  |
+
+Once these environment variables are set, you can start your worker:
+```bash
+make run
+```
+
+### Trick to set environment variables easily
+
+You could create a file named `.env` (or you can copy the file `.env.dist`) end edit it with the correct values.
+The `make run` command will automatically take it into account if it exists.
+
+## Makefile targets
+
+Commands below will be used for both stacks (backend & workers):
+
+| Command                     | Description                                              |
+|-----------------------------|----------------------------------------------------------|
+| `make build`                | Build the application                                    |
+| `make ci-code-format`       | Check the code format according to the rust format rules |
+| `make ci-code-coverage`     | Launch tests and returns code coverage for tests         |
+| `make ci-lint`              | Launch the rust linter                                   |
+| `make ci-tests`             | Launch tests                                             |
+| `make docker-build`         | Build locally a docker image                             |
+| `make docker-clean`         | Remove locally the built docker image                    |
+| `make docker-push-registry` | Push the locally built docker image                      |
+| `make run`                  | Run locally the worker                                   |
+| `make version`              | Display the version defined in `cargo.toml` file         |
+
+## CI / CD
+
+A `.gitlab-ci.yml` file is provided for the gitlab CI/CD feature.
+This file will instantiate te following pipeline:
+
+<!-- language: lang-none -->
+    /-----------\      /---------\             /----------\             /----------\
+    |  Compile  |------|  Tests  |-------------| Quality  |-------------|  Docker  |
+    \-----------/      \---------/             \----------/             \----------/
+         |                  |                       |                        |
+     +------+           +-------+              +---------------+          +-------+
+     | lint |           | tests |              | code-coverage |          | build |
+     +------+           +-------+              +---------------+          +-------+
+                                                    |
+                                               +-------------+
+                                               | code-format |
+                                               +-------------+
+<!-- language: markdown -->
+
+### Docker
+
+The command `make docker-build` will build an image named `ftvsubtil/rdf_worker`.
```

Comment:
the image will not be in `ftvsubtil` group

Reply:
Done.